scripts/build_test.py

```diff
@@ -1141,3 +1155,117 @@ def mock_get_file_count(unused_path: str) -> int:
                 AssertionError, 'webpack_bundles should be non-empty.'
             ):
                 build.build_using_webpack(build.WEBPACK_PROD_CONFIG)
+
+
+class E2EBuildTests(test_utils.GenericTestBase):
```

Comment:
This name suggests that these build methods are e2e-test-specific, but they aren't, right?

Reply:
@U8NWXD Actually they are. Since we have shifted some methods directly from `run_e2e_tests.py` to build.py. It was done because we required the same methods in `run_acceptance_tests.py`. Also these methods are build specific as you may see in the test names.

Reply:
Sorry I wasn't clear. I meant that since they're used in both the e2e and the acceptance tests, they're not specific to the e2e tests. To me, the current name implies that they're only used in the e2e tests, not the acceptance tests

Reply:
Ohh yes absoulutely right. We can change the naming to something like e2eAndAcceptanceBuildTests or something else